procrastinate/app.py

```diff
@@ -151,6 +157,7 @@ def _wrap(func: Callable[..., "tasks.Task"]):
                 app=self,
                 queue=queue,
                 name=name,
+                aliases=aliases,
```

Comment:
Do we really need to store the aliases in the Task?

Reply:
Good remark. This is most probably not needed.

Reply:
During a previous discussion, we thought it was a good idea to have a Task access to its aliases. But I have no problem to remove it if you think it's useless.

Reply:
Actually it's also what allows to register the task aliases in `_register`. How should we handle it if we don't store aliases in Task ?

Reply:
You can pass the aliases to the `_register` function, no?

Reply:
As discussed with @elemoine, my point is that it could be useful for later introspection and good for staying consistent with the fact we already store the task's name.
That's not a blocking point for him, so I'll keep like this :) Thanks @elemoine for your input.

Reply:
I also said that I am not a big fan with storing the aliases in the `Task` object for the sole purpose of passing them to the `_register` function. But it may be useful to have the aliases in the `Task` object for other purposes. Anyway, not a blocker for me.